bbn-test-5/babylon-node/README.md

```diff
@@ -0,0 +1,283 @@
+correct point. You have two options:
+
+1. Download directly from the RPC endpoint: `https://rpc.devnet.babylonlabs.io/genesis`
+
+2. Or use these commands in your terminal:
+
+```shell
+wget https://github.com/babylonlabs-io/networks/raw/main/bbn-test-5/genesis.tar.bz2 # TODO: update this file name if necessary
+tar -xjf genesis.tar.bz2 && rm genesis.tar.bz2
+mv genesis.json ~/.babylond/config/genesis.json
+```
+
+>Note: Verify that the `chain-id` in the genesis file matches the one used in 
+your initialization command (`bbn-test-5`). This ensures your node connects 
+to the correct network.
+
+## Setup the required keys for operating a validator 
+
+### Keys for a CometBFT validator
+
+The validator key is a fundamental component of your validator's identity 
+within the Babylon network. This cryptographic key-pair serves multiple critical 
+functions: it signs blocks during the consensus process, validates transactions, 
+and manages your validator's operations on the network. Creating and securing this 
+key is one of the most important steps in setting up your validator.
+
+To generate your validator key, use the following command:
+
+```shell
+babylond --keyring-backend test keys add <name> --home <path>
+```
+We use `--keyring-backend test`, which specifies which backend to use for the 
+keyring, `test` stores keys unencrypted on disk. 
+
+There are three options for the keyring backend:
+
+`test`: Stores keys unencrypted on disk. It’s meant for testing purposes and 
+should never be used in production.
+`file`: Stores encrypted keys on disk, which is a more secure option than test but 
+less secure than using the OS keyring.
+`os`: Uses the operating system's native keyring, providing the highest level of 
+security by relying on OS-managed encryption and access controls.
+
+The `<name>` specifies a unique identifier for the key.
+
+The execution result displays the address of the newly generated key and its 
+public key. Following is a sample output for the command:
+
+```shell
+- address: bbn1kvajzzn6gtfn2x6ujfvd6q54etzxnqg7pkylk9
+name: <name>
+pubkey: '{"@type":"/cosmos.crypto.secp256k1.PubKey",
+key: "Ayau+8f945c1iQp9tfTVaCT5lzhD8n4MRkZNqpoL6Tpo"}'
+type: local
+```
+
+Make sure to securely store this information, particularly your address and 
+private key details. Losing access to these credentials would mean losing 
+control of your validator and any staked funds.
+
+### Keys for a BLS Voting
+#### What is BLS Voting
+
+BLS (Boneh-Lynn-Shacham) voting is used to create compact checkpoints at the 
+end of each epoch. An epoch is a fixed period within the blockchain, defined or 
+set by a number of blocks, during which a validator set remains consistent. 
+This stability allows Babylon to create and submit checkpoints only once per 
+epoch, reducing the overhead of interacting with the Bitcoin blockchain.
+
+To read more on BLS Voting please see [here](#) # TODO: add link when it is ready 
```

Comment:
No need to have this if we don't have the blog yet. We can comment this out entirely.

Also, `#` comments don't work on markdown.